draft-ietf-rats-corim.md

```diff
@@ -908,11 +908,17 @@ The following describes each member of the `class-map`:
 
 ##### Environment Instance {#sec-comid-instance}
 
-An instance carries a unique identifier that is reliably bound to a Target Environment
-that is an instance of the Attester.
+An `instance-id` is a globally unique value that identifies a Target Environment instance.
+The identifier is reliably bound to the Target Environment.
+For example, if the Target Environment is a DICE layer {{DICE.Layer}} the `instance-id` might be an X.509 subjectPublicKey.
+See ({{Section 4.1 of -pkix-cert}}).
+The subjectPublicKey is reliably bound to the DICE layer because the key generation is tied to a measurement of the Target Environment.
+Alternatively, if the Target Environment is an X.509 subjectPublicKey the `instance-id` might be a key identifier that is a digest of the public key.
+See {{Section 4.2.1.2 of -pkix-cert}}.
+The keyIdentifier is reliably bound to the subjectPublicKey because the identifier is a digest of the key.
```

Comment:
By `keyIdentifier` do we mean the `SubjectKeyIdentifier` extension?  Or is it the `KeyIdentifier` ASN.1 type from RFC5280?